lessons/Data-Science/measures-of-center/langs/en-us/index.adoc

```diff
@@ -35,150 +36,110 @@
-- In each of the first 3 Investigate sections of this lesson, students will calculate a Measure of Center from a list. If you would like to be able to direct students to printed versions with space for them to write, make copies of @opt-printable-exercise{mean-median-modes.adoc}.
-- If you are using our Google Slides, adjust them based on which of these activities you will be doing with your students.
+- The first section of the lesson is an introduction to measures of center. If you are confident that your students have a solid understanding of mean, median, and mode, you can jump to the second section of the lesson, where students will use @proglang to compute measures of center, and then consider which measure of center is the most appropriate.
+- If you are using our Google Slides, adjust the slides based on which portions of the lesson you will be doing with your students.
 }
 
 |===
 
-== Mean @duration{15 minutes}
+== Introduction to Measures of Center @duration{15 minutes}
 
 === Overview
-Students learn about @vocab{mean} (or "average") as one way (among others!) to summarize a @vocab{quantitative} column, and how to compute it using @proglang.
+
+Students learn how to compute three different measures of center.
 
 === Launch
 
-One of the ways that Data Scientists summarize quantitative data is by talking about its _center_ - literally asking "what is a typical value in this sample?", in the hopes of inferring something about a larger population.  
+@teacher{Using the prompt below, invite your students to share what they know about mean, median, and mode. If your students need to review these concepts, continue to teach the entirety of this lesson section to provide instruction and practice. If your students are confident with computing mean, median, and mode, skip to the next section of the lesson, where students will (1) use Pyret to compute various measures of center and (2) begin considering the usefulness of each measure of center.}
 
-But there are many different ways to define "center", and each method has strengths and weaknesses. The shape of the data can play a huge role in whether or not one kind of summary is appropriate!
 
-@slidebreak
+In our work with dot plots and histograms, we often asked ourselves, "What is a typical value in this sample?"
 
-@lesson-instruction{
-Let's take a moment to consider what values might be typical for the weight of our animals by completing @printable-exercise{typical.adoc}.
-}
+When we posed this question, we often discovered that different students in our class had a different responses!
 
-@slidebreak 
+- Some of you looked for the value that was most commonly repeated.
+- Some of you looked for the midpoint of the sample.
+- Some of you searched for an appropriate "balance point" among the data points.
 
-@center{@image{images/pounds.png}}
+It turns out that these different ways of thinking about typicality are all different ways of "measuring the center".
```

Comment:
This lesson never mentions "Central Tendency", despite it being all about central tendency. It's never bothered me much before and it still doesn't, but this sentence and our focus on variability makes me wonder if this isn't the place to tie them all together: "what value is typical?" is asking "is there a value that all the others _tend to cluster around_?"

Just a thought.

Reply:
Could I just add what you wrote? 

It turns out that these different ways of thinking about typicality are all different ways of "measuring the center".  "What value is typical?" is asking "is there a value that all the others tend to cluster around?"

...or do you actually want me to throw in the word "central tendency"? If you have a sentence or two to suggest, I'd be grateful.